portia-entrypoint

```diff
@@ -80,11 +111,54 @@ def download_portia_bundle(bundle_url, jobkey, jobauth):
     return bundle_req.content
 
 
+def unpack_eggs_bundle():
+    if not os.path.isdir(DASH_PACKED_EGGS_PATH):
+        os.mkdir(DASH_PACKED_EGGS_PATH)
+    try:
+        zf = ZipFile(DASH_BUNDLE_PATH)
+        zf.extractall(DASH_PACKED_EGGS_PATH)
+    except BadZipfile as exc:
+        _print_flush("Bad zipfile %s" % exc)
+    else:
+        uid = int(os.environ.get('SHUB_JOB_UID', 65534))
+        for eggname in os.listdir(DASH_PACKED_EGGS_PATH):
+            eggpath = os.path.join(DASH_PACKED_EGGS_PATH, eggname)
+            if os.path.isfile(eggpath) and eggname.endswith('.egg'):
+                os.chown(eggpath, uid, GID)
+
+
+def load_dash_eggs(eggs_path):
+    pypath = os.environ.get('PYTHONPATH', '')
+    for egg_path in glob.glob("%s/*.egg" % eggs_path):
+        if is_zipunsafe(egg_path):
+            egg_path = unpack_egg(egg_path)
+        sys.path.insert(0, egg_path)
+        pypath += '%s:' % egg_path
```

Comment:
if pypath is not empty and doesn't end with colon you'll get a malformed pythonpath here. also I think u may want dash eggs to be at the beginning of the path, not appended to the end